pages/linux/auracle.md

```diff
@@ -0,0 +1,20 @@
+# auracle
+
+> Auracle is a command line tool used to interact with Arch Linux's User Repository, commonly referred to as the AUR.
+> More information: <https://github.com/falconindy/auracle>.
+
+- Find packages in the AUR by regular expression:
+
+`auracle search {{regexp}}`
+
+- Print detailed information about packages:
+
+`auracle info {{packagename}} {{packagename}}`
+
+- Show the contents of a source file for a/multiple package(s) (e.g. the PKGBUILD):
```

Comment:
The `PKGBUILD` is not an example, because the command only show the `PKGBUILD` content, maybe this is better:
```suggestion
- Display the  `PKGBUILD` file (build information) for a space-separated list of AUR packages:
```